### DISCUSSION TITLE
Flutter Developers

### DISCUSSION THREAD

**Discussion by Eng-Mouaz-M-AlShahmeh** [community/events#19] [AI]
What about Github Copilot Updates?

**Comment by Akash1134** [comment]
Hi @Eng-Mouaz-M-AlShahmeh 
Thanks for joining us for the GitHub Universe celebrations in the GitHub Community, we’re glad you’re here!
Please refer to this [blog post](https://github.blog/2022-11-09-everything-new-from-github-universe-2022/#redefine-whats-possible) with all the details about Copilot updates shipped at the Universe.
